bin/pyrdp-mitm.py

```diff
@@ -50,10 +50,11 @@ def main():
     parser.add_argument("--crawler-ignore-file", help="File to be used by the crawler to chose what folders to avoid when scraping the client shared drives.", default=None)
     parser.add_argument("--no-replay", help="Disable replay recording", action="store_true")
     parser.add_argument("--no-downgrade", help="Disables downgrading of unsupported extensions. This makes PyRDP harder to fingerprint but might impact the player's ability to replay captured traffic.", action="store_true")
+    parser.add_argument("--test", help="Test the initialization of the program. Initialize the MITM, then exit without listening for connections.", action="store_true")
 
     args = parser.parse_args()
     outDir = Path(args.output)
-    outDir.mkdir(exist_ok = True)
+    outDir.mkdir(exist_ok=True)
```

Comment:
You couldn't resist :wink:

Reply:
i see pep8 violation i need to act 😤

Reply:
I thought it was okay if its the same file though 😂

Reply:
It's within diff distance (-3/+3 of a related change) so it's good. Same file is too large but diff distance is ok. My comment was a joke. The change should stay in.